### PR TITLE
Clean up the documentation (#92)

### PR DESCRIPTION
**Pull request by agarny**
Addresses issue #92.

### CODE REVIEW

**Comment by agarny** [comment]
I am confused:
- The first time my changes [failed](http://autotest.bioeng.auckland.ac.nz/libcellml-buildbot/builders/Documentation%20Builder/builds/177/steps/documentation/logs/stdio) is because the Sphinx theme directory was hard-coded as opposed to being generic (my bad!).
- However, after [fixing the problem](https://github.com/cellml/libcellml/pull/93/commits/e38ca82f118ebf5b90d416a23b495a447901a119), I was hoping that it would all be fine, but now Sphinx tells me that it [cannot find](http://autotest.bioeng.auckland.ac.nz/libcellml-buildbot/builders/Documentation%20Builder/builds/178/steps/documentation/logs/stdio) the `Alabaster` theme?! This is the theme that we have been using for some time, so why that error? @hsorby?

File: README.rst [hunk 1]
```diff
@@ -28,7 +28,7 @@ Continuous Integration
 ----------------------
 
 libCellML makes use of Buildbot for continuous integration testing, all code is run through the
-test suite when a pull request is created or added to.  More information on how Buildbot is 
+test suite when a pull request is created or added to.  More information on how Buildbot is
 integrated in to the development process is available from the documentation http://libcellml.readthedocs.org/.
```
Comment (nickerso) [inline README.rst]: another link to update.

Comment (agarny) [inline README.rst]: Hmm, I wonder how I managed to miss that one (and a couple more), but thanks for the catch!

**Comment by dladd** [comment]
> However, after fixing the problem, I was hoping that it would all be fine, but now Sphinx tells me that it cannot find the Alabaster theme?

I just had a look at this and was able to get it building by downloading and extracting the [alabaster theme](https://github.com/bitprophet/alabaster) into the docs directory. Maybe we should set the theme up as a submodule?

**Comment by nickerso** [comment]
Its strange, cause for me it builds out of the box (as it does for @agarny). I'm using Sphinx 1.3.6 if that makes a difference - which it might since alabaster is included in sphinx since 1.3...

**Comment by dladd** [comment]
That will probably be the problem- I'm on Ubuntu 14.04 (as is the buildbot) and my apt-installed Sphinx is 1.2.2 . So maybe we need to upgrade Sphinx or install the theme on the buildbot? Although this still doesn't explain why this hasn't been an issue previously.

**Comment by nickerso** [comment]
buildbot seems to be using 1.3.1 (http://bioeng1030.bioeng.auckland.ac.nz:8101/963175d9f01efc3c0abf5b7731fe41b4136b1843/dox/) - which does have the alabaster theme... ???

**Comment by hsorby** [comment]
I have upgraded the version of Sphinx in use on the build machine.  It appeared that that particular version of sphinx had a bug when extending either alabaster or rtd_themes.  The documentation should now build successfully for libcellml.

**Comment by nickerso** [comment]
thanks @hsorby - I re-triggered the build and everything is passing now and the generated dox seem to be correct (i.e., the dotted line is missing!): [see here](http://autotest.bioeng.auckland.ac.nz/libcellml-buildbot/43c2a286b68e3f03ae7b3316e704a19dbe572391/dox/).

I'm happy for this to be merged in.

**Comment by agarny** [comment]
Brilliant, thanks a lot @hsorby.

**Comment by dladd** [comment]
I'm not sure if I get a vote but I think this ready to merge.

**Pull request merged**